pkgs/development/perl-modules/generic/default.nix

```diff
@@ -20,6 +20,8 @@ perl.stdenv.mkDerivation (
     # authors to skip certain tests (or include certain tests) when
     # the results are not being monitored by a human being."
     AUTOMATED_TESTING = true;
+
```

Comment:
I have not tried to run it, but at the first glance it looks like that any ```meta``` (say ```meta.license``` from ```attrs``` below would destroy this ```meta.homepage```

Reply:
Oh, yep, it does :(

Reply:
I happened to try testing a package that only had that one meta attribute.